src/decisionengine/framework/dataspace/dataspace.py

```diff
@@ -174,8 +174,8 @@ def mark_demented(self, taskmanager_id, keys, generation_id=None):
     def close(self):
         self.datasource.close()
 
-    def store_taskmanager(self, name, id):
-        return self.datasource.store_taskmanager(name, id)
+    def store_taskmanager(self, name, taskmanager_id, datestamp=None):
```

Comment:
No code calls this by the old names.  This pattern makes it match the `datasource.store_taskmanager` api.